### PR TITLE
Fixes hound issues for %x

### PR DESCRIPTION
**Pull request by sayalilunkad**
Replaces %x with backticks as per updated
ruby style.

### CODE REVIEW

File: chef/cookbooks/swift/recipes/proxy.rb [hunk 1]
```diff
@@ -271,7 +271,7 @@
             end
 
             # Generate self-signed certificate with above CSR
-            %x(openssl x509 -req -days 3650 -in #{ssl_csr_file} -signkey #{node[:swift][:ssl][:keyfile]} -out #{node[:swift][:ssl][:certfile]})
+            `(openssl x509 -req -days 3650 -in #{ssl_csr_file} -signkey #{node[:swift][:ssl][:keyfile]} -out #{node[:swift][:ssl][:certfile]})`
```
Comment (aplanas) [inline chef/cookbooks/swift/recipes/proxy.rb]: Not Ruby expert, but I think that you need to replace the full `%x()`, including the parents (the same for the rest of the lines)

**Comment by aplanas** [comment]
Not sure if we are using this kind of refactoring PR to also address the long line complains.

+1 when gating is green

**Comment by sayalilunkad** [comment]
@aplanas there is currently a discussion on the cloud-devel ML about if we want to increase the line width to silence that error. Maybe after there is some conclusion on that we can act on it.

**Comment by rsalevsky** [comment]
+1

File: chef/cookbooks/tempest/recipes/config.rb [hunk 2]
```diff
@@ -266,7 +266,7 @@
 
 # FIXME: the command above should be good enough, but radosgw is broken with
 # tempest; also should avoid search with no environment in query
-#%x{#{keystone} endpoint-get --service object-store &> /dev/null}
+#%x{#{keystone} endpoint-get --service object-store &> /dev/null}`
```
Comment (rsalevsky) [inline chef/cookbooks/tempest/recipes/config.rb]: # `#{keystone} endpoint-get --service object-store &> /dev/null}`

**Pull request merged**